src/Bounds.cpp

```diff
@@ -733,6 +733,11 @@ class Bounds : public IRVisitor {
         }
     }
 
+    void visit(const Shuffle *op) {
+        // TODO: We could consider the bounds of the lanes used by the Shuffle.
```

Comment:
This might generate gigantic bounds expressions though. In Hexagon, vectors with 512 lanes are not unlikely (e.g. an expression that converts 8 bit to/from 32 bit will need to vectorize 4x the native vector width).

I'd also like to get this in where it replicates the old behavior exactly (which is what this does, the old Call visitor didn't do anything with shuffle intrinsics), and then make further improvements. Can we leave it as a TODO for now and come back to this?